js/index.js

```diff
@@ -1,6 +1,36 @@
-// Iteration 1: Names and Input
-//
-// Iteration 2: Conditionals
 
+// Iteration 1
+let hacker1 = "luisa"
+console.log("The driver's name is", hacker1);
+let hacker2 = "majid"
+console.log("The navigator's name is", hacker2);
+ 
 
-// Iteration 3: Loops
+// Iteration 2
+if(hacker1.length > hacker2.length) {
+  console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+} else if (hacker1.length < hacker2.length) {
+  console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+} else {
+  console.log('Wow, you both have equally long names!');
+}
+
+
+// Iteration 3
+let capitalName = "";
+let lowercaseName = "";
+
+for(let i = 0; i< hacker1.length; i++) {
+  console.log(hacker1[i]);
+}
+for(let i = hacker2.length-1; i>= 0; i--) {
+  console.log(hacker2[i]);
+}
```

Comment:
Here it is not working as expected because when you console.log() inside of a loop it will print each character at a time in a new line, meaning that you would need to store the name in a variable and then console.log it like this: 
```
let newName = ""
for (let i = 0; i < hacker1.length; i++) {
   newName += hacker1[i].toUpperCase() + " ";
}
console.log(newName)
```
So, here on top we are creating a variable and we are adding each letter from the hacker1 name one by one with a space in between. Meaning that when we console.log it will all be in the same line as expected